corehq/apps/app_manager/models.py

```diff
@@ -2030,6 +2030,10 @@ def from_json(cls, data):
                     item.value[lang] = interpolate_media_path(path)
         return to_ret
 
+    @property
```

Comment:
I don't think this is adding much value, since we already expose the string values for detail column formats outside of models.py (app_strings.py and detail-screen-config.js use them).